src/pages/Register.js

```diff
@@ -57,30 +62,44 @@ export const Register = () => {
       })
   }
   return (
-    <form onSubmit={onFormSubmit}>
-      <label htmlFor="username"> Username </label>
-      <input
-        type="text"
-        id="username"
-        value={username}
-        onChange={(e) => setUsername(e.target.value)} />
-      <label htmlFor="email"> Email </label>
-      <input
-        type="email"
-        id="email"
-        value={email}
-        onChange={(e) => setEmail(e.target.value)} />
-      <label htmlFor="password"> Password </label>
-      <input
-        type="password"
-        id="password"
-        value={password}
-        onChange={(e) => setPassword(e.target.value)} />
-      <button type="submit">Register</button>
-      <button
-        type="button"
-        onClick={onBackClick}> Back
-      </button>
-    </form>
+    <section className="register-section">
+      <p>Hi! Start your journey here</p>
+      <h1 className="register-title">Create account</h1>
+      <div className="register-fields">
+        <form onSubmit={onFormSubmit}>
+          <input
+            type="text"
+            id="username"
+            placeholder="Username"
+            value={username}
+            onChange={(e) => setUsername(e.target.value)} />
+          <input
+            type="email"
+            id="email"
+            placeholder="Email"
+            value={email}
+            onChange={(e) => setEmail(e.target.value)} />
+          <input
+            type="password"
+            id="password"
+            placeholder="Password"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)} />
+          <button className="register-btn2" type="submit">Register</button>
+          {/* <button
+            type="button"
+            onClick={onBackClick}> Back
+          </button> */}
```

Comment:
We should decide what to do with the "bacl button" across the entir eapplication, if the user refreshes the page they get logged out right now